app/main.py

```diff
@@ -21,6 +21,11 @@ class Transaction(BaseModel):
     price: int
     currency_inserted: int
 
+class UpdateTransaction(BaseModel):
+    item_name: Optional[str] = None
+    price: Optional[int] = None
+    currency_insterted: Optional[int] = None
+
 @app.get("/")
 def read_root():
     return {"Hello": "Please enter your item."}
```

Comment:
It seems to me that we could be a little more specific about whats going on for our keys. For example, when someone shows up at the root, we could maybe return {"Successful Login": "Hello, please enter your item."}. We currently have a couple different error messages with the key "Error", so this would also help clarify which error message we are looking for.

Reply:
So I am thinking of going the other way with it. Because key traversal isn't the best way to do things IMO. 

What if we did something like this: 
```
{
  "status": "Successful Login",
  "message:": "Hello, please enter your item"
}
```

```
{
  "status": "ERROR: Item Not Found",
  "message:": "Sorry, We couldn't find that item."
}
```

By changing our structure as such we can then extract the status key always to determine what happened, this could be standard across all of our apps, same with the messaging. We would need zero special logic to display the message at any point.

Reply:
Oh, ok. I think I see where you're coming from. Let me make some tweaks and push them to see what you think.